lisa/sut_orchestrator/azure/tools.py

```diff
@@ -63,12 +63,14 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
 
     def get_version(self) -> str:
         result = self.run("-version")
-        if isinstance(self.node.os, CBLMariner):
-            self._command = "/usr/bin/waagent"
-        else:
-            self._command = "/usr/sbin/waagent"
+
         if result.exit_code != 0:
+            if self.node.tools[Ls].path_exists("/usr/bin/waagent", sudo=True):
```

Comment:
I think sudo is not needed. From original code, even `sbin` is used, no sudo is used for "-version" later.

Reply:
removed, will test